networking.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: page
+title: Making Networking Requests in Flutter
+permalink: /networking/
+---
+
+* TOC Placeholder
+{:toc}
+
+## Using the `http` package
+
+Flutter supports the the [http package][http], version `0.11.3+12` or later.
+
+Start by declaring a dependency by adding an http line under `dependencies` in
+`pubspec.yaml`:
+
+```
+dependencies:
+  ...
+  http: '>=0.11.3+12'
+```
+
+## Making requests
```

Comment:
If this is a sub-header, please use ### , otherwise, we can be more specific in this header: "Making HTTP requests".  (to clarify: which kind of requests?)